### PR TITLE
Update survey form

### PR DESCRIPTION
**Pull request by decrn**
image

Added placeholder actions, still up for debate and status icons (they have tooltips!)

### CODE REVIEW

File: src/common/components/status-tag/status-tag.component.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import React, { Component } from 'react';
+import { mapStatusToColor } from '../../mappers/survey-status.mapper';
+import { SurveyStatus } from '../../types/survey-status.type';
+
+export interface StatusTagProps {
+    status: SurveyStatus;
+}
+
+export class StatusTag extends Component<StatusTagProps> {
+    private mapStatusToTooltip(status: SurveyStatus): string {
```
Comment (kfrancois) [inline src/common/components/status-tag/status-tag.component.tsx]: These should be arrow functions ideally

File: src/containers/homepage/survey-list.component.tsx [hunk 2]
```diff
@@ -36,7 +45,23 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 key: 'questionCount',
                 render: (text: any) => text,
             },
+            {
+                title: 'Actions',
+                render: () => (
+                    <>
```
Comment (kfrancois) [inline src/containers/homepage/survey-list.component.tsx]: This doesn't seem right

Comment (decrn) [reply src/containers/homepage/survey-list.component.tsx]: https://reactjs.org/docs/fragments.html

Comment (kfrancois) [reply src/containers/homepage/survey-list.component.tsx]: I'm baffled

**Pull request merged**